scripts/check-health.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+# assert on the health of a running container
+IS_SYNCING=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.isSyncing')
+SHOULD_HAVE_PEERS=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.shouldHavePeers')
+PEERS=$(curl --silent http://127.0.0.1:9933/health | jq  -r '.peers')
+
+if [ "$IS_SYNCING" != "true" ]; then
```

Comment:
`isSyncing` indicates that a node is catching up on missed blocks (more precisely it's true when a node is >5 blocks behind the network's best block), so it's expected to be false when a node is fully synced and running properly.